io/common/include/detray/io/payloads.hpp

```diff
@@ -42,7 +45,8 @@ struct grid_payload {
 
 /// @brief A payload object for material
 struct material_slab_payload {
-    std::array<real_io, 5u> slab;
+    std::array<real_io, 8u> slab;
+    std::array<real_io, 7u> density_eff;
```

Comment:
Did you take out density effect data because ACTS doesn't have it yet?

Reply:
Not so sure what to do with that, tbh. But right now, there is no constructor in detray without the density data and I wanted to write the json as detray-native as possible to see where we are

Reply:
Also, I think it would make sense to separate the geometry json from the material json, so that we can switch to material maps more seamlessly?